src/encoder.rs

```diff
@@ -3242,12 +3256,16 @@ fn encode_tile_group<T: Pixel>(
       let rec = &mut ts.rec;
       cdef_filter_tile(fi, &deblocked_frame, &blocks.as_tile_blocks(), rec);
     }
-    /* TODO: Don't apply if lossless */
-    fs.restoration.lrf_filter_frame(
-      Arc::get_mut(&mut fs.rec).unwrap(),
-      &deblocked_frame,
-      fi,
-    );
+    // SAFETY: We know no other threads are accessing this because we are done with tiles.
+    // The only references would be in ref frames, which are not being touched at the moment.
```

Comment:
What is the gain from doing this? Maybe it should be moved to a different patch.